[ADD] events: added booth doc to 16 (#10330)
odoo/documentation: content/applications/marketing/events/event_booths.rst

============
Event booths
============

> The Odoo *Events* application provides users with the ability to create event booths, sell their

Reviewer: ```suggestion
The Odoo ***Events*** application provides users with the ability to create event booths, sell their
```
App name in bold

availability, and manage their reservations.

Configuration
=============

In order to create, sell, and manage booths for events, the *Booth Management* feature **must** be
activated.

To do that, navigate to :menuselection:`Events app --> Configuration --> Settings`, and in the
:guilabel:`Events` section, tick the :guilabel:`Booth Management` checkbox. Then, click
:guilabel:`Save`.

> .. image:: event_booths/booth-management-setting.png

Reviewer: Switch to a figure and remove the markup on image

   :align: center
   :alt: The Booth Management setting in the Odoo Events application.

.. important::
   When the :guilabel:`Booth Management` setting is activated, a new *Product Type* becomes
   available on all product forms: *Event Booth*.

>    This is important because every created booth **must** be assigned a *Booth Category* on their

Reviewer: ```suggestion
   This is important because every created booth **must** be assigned a *Booth Category* on its
```

   respective booth form, and every booth category **must** have an *Event Booth* product assigned
   to it.

Booth categories
================

> With the *Booth Management* setting activated in the *Events* app, the *Booth Categories* option

Reviewer: ```suggestion
With the *Booth Management* setting activated in the **Events** app, the *Booth Categories* option
```

  user.
- :guilabel:`Product`: the *Event Booth* product associated with that specific booth category.
- :guilabel:`Price`: the price of a booth in that booth category.

When the :icon:`oi-settings-adjust` :guilabel:`(settings)` icon, located to the far-right of the
column titles, is clicked, a drop-down menu of additional column options appears. From the resulting
drop-down menu, tick the checkbox beside :guilabel:`Sponsor Level` and/or :guilabel:`Sponsor Type`
to reveal those columns on the :guilabel:`Booth Category` page.

To edit an existing booth category, select it from the list, and proceed to make any desired
modifications from the event category form.

Create booth category
---------------------

To create a booth category from the :guilabel:`Booth Category` page, click the :guilabel:`New`
button in the upper-left corner to reveal a blank booth category form.

.. image:: event_booths/booth-category-form.png
   :align: center
   :alt: A typical booth category form in the Odoo Events application.

Start by entering a name for the booth category in the top :guilabel:`Booth Category` field. This is

> a **requried** field.

Reviewer: ```suggestion
a **required** field.
```

camera placeholder in the upper-right corner of the booth category form. When clicked, proceed to
upload the desired image to the booth category form, if needed.

In the :guilabel:`Booth Details` section, users **must** assign a :guilabel:`Product` to the
category, and it **must** have *Event Booth* set as the *Product Type* on the product form.

And, regardless of the listed price on the *Event Booth* product chosen, the user can input a custom
:guilabel:`Price` to be applied for this booth category in the field below.

In the :guilabel:`Sponsorship` section, there is a :guilabel:`Create Sponsor` checkbox option. With
that checkbox ticked, whenever a booth belonging to this category is booked, the user is created as
an official *Sponsor* of the event.

When the :guilabel:`Create Sponsor` checkbox is ticked, two additional fields appear beneath it:
:guilabel:`Sponsor Level` and :guilabel:`Sponsor Type`.

.. note::
   :guilabel:`Sponsor Level` and :guilabel:`Sponsor Type` are purely to distinguish different
   distinctions of sponsors.

   For example, if a sponsor has been attached to a company for multiple years, they would be

>    granted a higher level (e.g. *Gold* level), which provides them with immediate credability and

Reviewer: ```suggestion
   granted a higher level (e.g. *Gold* level), which provides them with immediate credibility and
```

   status. Whereas, conversely, a relatively new sponsor would be granted a lower level (e.g.

>    *Bronze* level), which coincides with its own credability and status.

Reviewer: ```suggestion
   *Bronze* level), which coincides with its own credibility and status.
```

   found on a common booth category form.

   Simply clicking :guilabel:`Create` from the resulting drop-down menu that appears after typing in
   a new name creates the category, but does **not** reveal the :guilabel:`Create Booth Category`
   pop-up window. The category would have to be modified later, via the *Booth Categories* page
   (:menuselection:`Events app --> Configuration --> Booth Categories`).

Upon selecting a pre-existing :guilabel:`Booth Category`, two additional, non-modifiable fields
appear: :guilabel:`Product` and :guilabel:`Price`. Both fields represent their respective selections
for that specific booth category.

When a person purchases a booth rental through the event website, the subsequent renter-related
fields on the form auto-populate, based on the information provided by the purchaser during the
online transaction. The booth also automatically changes its status from :guilabel:`Available` to
:guilabel:`Unavailable`.

However, if the rental of a booth is conducted in any other way (e.g., in person, via sales order,
etc.), the :guilabel:`Renter`, :guilabel:`Renter Name`, :guilabel:`Renter Email`, :guilabel:`Renter

> Phone`, and :guilabel:`Renter Mobile` fields can be entered in manually.

Reviewer: ```suggestion
Phone`, and :guilabel:`Renter Mobile` fields can be entered manually.
```

while in Kanban view.

Sell event booths
=================

With event booths configured on the event form, via the event-specific *Booths* pages, Odoo presents
them on the event website, via the *Get A Booth* event subheader link.

To access the *Get A Booth* page on the event website, open the :menuselection:`Events app`, and
select the desired event from the :guilabel:`Events` dashboard. From the event form, click the
:guilabel:`Go to Website` smart button to be taken to the Odoo-built event website.

If the event subheader menu (with the :guilabel:`Get A Booth` option) is *not* showing up on the
event website, there are two ways to make it appear.

While on the event website, enter the edit mode by clicking the :guilabel:`Edit` button in the
upper-right corner. Then, click into the :guilabel:`Customize` tab of the resulting sidebar of web
design tools.

In the :guilabel:`Customize` tab, click the toggle switch for :guilabel:`Sub-Menu (Specific)`, and
click :guilabel:`Save`. Doing so reveals the event subheader menu with various options.

Alternatively, enter :doc:`Debug mode <../../general/developer_mode>`, and open the specific event

> form in the the *Events* application.

Reviewer: ```suggestion
form in the *Events* application.
```

On the event form, with *Debug mode* on, an array of subheader menu options appears. Tick the
checkbox for :guilabel:`Website Submenu`, in order for the submenu to appear on the event website.
Doing so also ticks every other submenu-related checkbox automatically.

> At this point, proceed to choose which options to keep on the event subheader menu. In this case,

Reviewer: ```suggestion
At this point, choose which options to keep on the event subheader menu. In this case,
```

make sure the :guilabel:`Booth Register` checkbox is ticked.

On the event website, click the :guilabel:`Get A Booth` event subheader menu option. Doing so
reveals the :guilabel:`Get A Booth` page, showcasing all the configured event booths that were
created on the event form.

.. image:: event_booths/get-a-booth-page.png
   :align: center
   :alt: Typical Get A Booth page on the event website via the Odoo Events app.

From here, the visitor can select their desired booth option, then :guilabel:`Location`. Next, they
would click the :guilabel:`Book my Booths` button, located at the bottom of the :guilabel:`Get A
Booth` page.

Doing so reveals a :guilabel:`Sponsor Details` page, wherein they fill out either *Sponsor Details*
or *Contact Details*, depending on how the booth was configured on the event form. The fields

> present on this form vary, depending on whether its meant for a basic contact or an event sponsor.

Reviewer: ```suggestion
present on this form vary, depending on whether it's meant for a basic contact or an event sponsor.
```

.. note::
   If the selected booth has the *Create Sponsor* checkbox ticked, this page reads as *Sponsor
   Details*.

The information provided on this details page is used to auto-populate the renter-related
information on the booth form, accessible via the event form, in the *Events* application.

Once the necessary information has been entered, the booth purchaser then clicks the :guilabel:`Book

> my Booths` button at the bottom of the page, and proceeds to complete the typical checkout process.

Reviewer: ```suggestion
my Booths` button at the bottom of the page, and completes the typical checkout process.
```

> Upon a successful payment confirmation, that selected booth automatically moves to the *Unavailable*

Reviewer: ```suggestion
Upon successful payment confirmation, that selected booth automatically moves to the *Unavailable*
```
